src/node_contextify.cc

```diff
@@ -409,26 +402,43 @@ class ContextifyContext {
     Local<Object> proxy_global = PersistentToLocal(isolate,
                                                    ctx->proxy_global_);
 
-    bool in_sandbox = sandbox->GetRealNamedProperty(property).IsEmpty();
-    bool in_proxy_global =
-        proxy_global->GetRealNamedProperty(property).IsEmpty();
-    if (!in_sandbox || !in_proxy_global) {
-      args.GetReturnValue().Set(None);
+    bool in_sandbox =
+      sandbox->HasRealNamedProperty(ctx->context(), property).FromMaybe(false);
```

Comment:
I need to rewrite this to be a little less crazy now that, as of https://github.com/v8/v8-git-mirror/commit/7b24219346fbfdf2ca795ab6cb370676bf148cb9, GetRealNamedProperty returns Nothing for absent, which allows us to only use GetRealNamedProperty instead of both HasRealNamedProperty and GetRealNamedProperty.